torchao/quantization/quant_primitives.py

```diff
@@ -41,12 +41,18 @@ class MappingType(Enum):
     we'll use (-10.2, 10.2) as the range for floating point and map that to (-8, 7)
     e.g. scale = (10.2 - (-10.2)) / (7 - (-8))
 
+    SYMMETRIC_NO_CLIPPING_ERR is a variant of symmetric mapping, where the scale is the max of smin
```

Comment:
cc @gau-nernst wondering if this will help int8 training as well: https://github.com/pytorch/ao/blob/1ce7da941b311ef6ab416d9c7a22be20e65b7495/torchao/prototype/quantized_training/int8.py#L65

Reply:
Possibly, will need to test. All INT8 training papers I see use 127 for both +ve and -ve though.

One potential concern I foresee is about quantization speed. During training we keep doing quantization, so having a fast one is crucial. Again, will need to try out / profile to be sure 😄.

Reply:
actually it seems even without this change 127 can be achieved with setting quant_min/quant_max to be (-127, 127)

yeah we could benchmark perf if we decide to use AQT for int8 training